### DISCUSSION TITLE
Download Personal Access Token

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22870] [New to GitHub]
<p>I have created a Personal Access Token on Github but have been unable to find how to download it to my computer so I can use it. What do I need to do?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>As is stated in step 9 of the <a href="https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token" rel="nofollow noopener">help documentation about creating a Personal Access Token</a>:</p>
<blockquote>
<p>For security reasons, after you navigate off the page, you will not be able to see the token again.</p>
</blockquote>
<p>If you didn’t copy down the Personal Access Token when you created it, there is no way to access it. You will have to create a new one.</p>
